index.html

```diff
@@ -23,7 +23,7 @@
 
         <div class="header-container">
             <header class="wrapper clearfix">
-                <h1 class="title">h1.title</h1>
+                <h1 class="title">OLA MUNDO´}´´</h1>
```

Comment:
Please remove typo error in "OLA", write "HOLA" instead of "OLA" and remove the ramdon characters "´}´´"

Reply:
DONE